camel/agents/chat_agent.py

```diff
@@ -382,10 +382,10 @@ def step_completion(
                                               self.model)
 
         # Terminate when number of tokens exceeds the limit
-        if self.model_config.max_tokens is None:
-            self.model_config.max_tokens = 500
-        if num_tokens >= self.model_token_limit + self.model_config.max_tokens:
-            return self.step_token_exceed(num_tokens)
+        assert isinstance(self.model_config, ChatGPTConfig)
```

Comment:
Asserts are not allowed in the package code.

Reply:
Thanks, done : )